pkg/export/otel/traces.go

```diff
@@ -376,11 +375,8 @@ func getTraceSettings(ctxInfo *global.ContextInfo, in trace.SpanExporter) export
 	}
 	meterProvider := metric.NewMeterProvider()
 	telemetrySettings := component.TelemetrySettings{
-		Logger:        zap.NewNop(),
-		MeterProvider: meterProvider,
-		LeveledMeterProvider: func(_ configtelemetry.Level) metric2.MeterProvider {
```

Comment:
Do you mind providing a bit of detail why we don't need this anymore? I think this was ported over from Alloy to Beyla by @marctc.